src/core/utils/addons.js

```diff
@@ -62,3 +63,51 @@ export const getFileHash = ({
 
   return undefined;
 };
+
+export function removeUndefinedProps(object: Object): Object {
+  const newObject = {};
+  Object.keys(object).forEach((key) => {
+    if (typeof object[key] !== 'undefined') {
+      newObject[key] = object[key];
+    }
+  });
+  return newObject;
+}
+
+export const getAddonJsonLinkedData = ({
+  addon,
+}: {|
+  addon: AddonType,
+|}): Object | null => {
+  const { current_version: currentVersion, ratings } = addon;
+
+  let aggregateRating;
+  if (ratings) {
```

Comment:
it wasn't clear to me in the issue if we should always be adding this or only if had a min amount of reviews (or rating too it mentioned).

Reply:
Yeah, it is not clear what to do. My understanding is that we should not round numbers and not hide/show numbers from time to time, i.e. always showing the ratings. That's what Google does.